package.json

```diff
@@ -74,61 +74,47 @@
 		]
 	},
 	"dependencies": {
-		"content-scripts-register-polyfill": "^4.0.2",
 		"webext-content-scripts": "^2.7.0",
-		"webext-detect": "^5.0.2",
-		"webext-events": "^3.0.1",
+		"webext-detect": "^5.3.2",
+		"webext-events": "^3.1.1",
 		"webext-patterns": "^1.5.0",
-		"webext-permissions": "^3.1.3",
+		"webext-permissions": "^3.1.4",
 		"webext-polyfill-kinda": "^1.0.2"
 	},
 	"devDependencies": {
-		"@parcel/config-webextension": "^2.12.0",
-		"@sindresorhus/tsconfig": "^6.0.0",
-		"@types/chrome": "^0.0.268",
+		"@parcel/config-webextension": "^2.13.3",
+		"@sindresorhus/tsconfig": "^7.0.0",
+		"@types/chrome": "^0.0.291",
 		"@types/firefox-webext-browser": "^120.0.4",
-		"@types/jest": "^29.5.12",
-		"expect-puppeteer": "^10.0.0",
+		"@types/jest": "^29.5.14",
+		"expect-puppeteer": "^11.0.0",
 		"jest": "^29.7.0",
 		"jest-chrome": "^0.8.0",
-		"jest-puppeteer": "^10.0.1",
+		"jest-puppeteer": "^11.0.0",
 		"npm-run-all": "^4.1.5",
-		"parcel": "^2.12.0",
-		"puppeteer": "^21.3.6",
-		"typescript": "^5.5.2",
-		"vitest": "^1.6.0",
-		"xo": "^0.58.0"
+		"parcel": "^2.13.3",
+		"puppeteer": "^23.11.1",
+		"typescript": "^5.6.3",
+		"vitest": "^2.1.8",
+		"xo": "^0.60.0"
 	},
 	"@parcel/resolver-default": {
 		"packageExports": true
 	},
-	"alias": {
-		"these-are-just-mocks-for-parcel-tests": "yolo",
-		"webext-permissions": "./test/demo-extension/webext-permissions.js",
-		"webext-dynamic-content-scripts": "./source/"
-	},
```

Comment:
Related:

- https://github.com/parcel-bundler/parcel/issues/8920

In this case I was able to move these to sub-folders so they don't get published.